xgo.go

```diff
@@ -26,6 +26,7 @@ var outPrefix = flag.String("out", "", "Prefix to use for output naming (empty =
 var srcRemote = flag.String("remote", "", "Version control remote repository to build")
 var srcBranch = flag.String("branch", "", "Version control branch to build")
 var crossDeps = flag.String("deps", "", "CGO dependencies (configure/make based archives)")
+var dockerImage = flag.String("image", "", "Use a custom docker image")
```

Comment:
Please set the flag description to "Use custom docker image instead of official distribution". I'd like to emphasize it that if you change this, you're on your own :)